Cargo.toml

```diff
@@ -24,4 +24,4 @@ members = [
 	"open-runtime-module-library/utilities",
 ]
 
-exclude = ["node", "runtime"]
\ No newline at end of file
+exclude = ["node", "runtime", "client"]
```

Comment:
Is there a reason we'd want to exclude the client? Isn't that where these integration tests will go?

Reply:
Currently, the CI is setup to run tests with `cargo test --release` command for the whole workspace. Integration tests will fail since they need a special script to run tests (starting a node in background + running test in synchronous mode to avoid nonce error).

Reply:
I can modify the CI script to exclude `client` when running tests, and then add a special script for the `client`